src/locales/translations/ru.json

```diff
@@ -633,12 +633,12 @@
 	"harvest": {
 		"title": "Харвестинг",
 		"status": "Статус",
-		"blocksHarvested": "Добыто Блоков",
-		"feesEarned": "Заработанные комиссии",
+		"blocksHarvested": "Добыто блоков",
+		"feesEarned": "Награды",
```

Comment:
```suggestion
		"feesEarned": "Награда",
```

Reply:
Changes made to line 637 ✅ Done

@OlegMakarenko